client/src/components/form.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState } from "react";
+import {
+  Typography,
+  Container,
+  CssBaseline,
+  makeStyles,
+  Button,
+  Divider,
+} from "@material-ui/core";
+
+import { Link } from "react-router-dom";
+import TextInput from "./TextInput";
+
+const useStyles = makeStyles((theme) => ({
+  paper: {
+    marginTop: theme.spacing(8),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+  },
+  form: {
+    marginTop: theme.spacing(1),
+    width: "100%",
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center",
+    justifyContent: "center",
+  },
+  submit: {
+    marginTop: theme.spacing(4),
+    backgroundColor: "#69E781",
```

Comment:
These colors should actually be coming from the theme.js file. You should look at material-ui documentation on using the theme. This will allow you to change all buttons to have similar colors easily as well as other things like the textField